config.py

```diff
@@ -1,5 +1,6 @@
 from os import environ
 from enum import Enum
+from dotenv import load_dotenv
```

Comment:
The `dotenv` package is not currently a dependency of this plugin. I'm not sure we should add it as one. If the user does not have `dotenv` installed, this line will currently result in an error and CTFd will not launch.

Reply:
Looking at the `dotenv` package, it is not well documented or clearly maintained. I'd prefer not to use it.

https://pypi.org/project/dotenv/